src/interfaces/dependencies.ts

```diff
@@ -41,6 +41,7 @@ export interface DependencyTree {
   parent?: DependencyTree
   files?: string[]
   postmessage?: string
+  type?: string
```

Comment:
https://github.com/typings/core/blob/master/src/lib/dependencies.ts#L445
is the only place that `DependencyTree.type` is set. Others (npm, etc) does not set it, thus I mark it as optional.